src/simulator/vacuum_cleaner.py

```diff
@@ -107,3 +110,12 @@ def get_required_delta_t(self) -> float:
         delta_t = min(self.turn_on_hit_angle / self.turning_speed, self.reverse_on_hit_duration) / 10
         return delta_t
 
+    def get_base(self, b: base.Base):
+        self.bases[b.oid] = b
+
+    def distance(self, oid: object.ObjectId) -> float:
+        """
+        robot get distance between self and base
+        """
+        if oid in self.bases:
```

Comment:
add an else which raises an exception (throws an error) saying that the given base id is not available or something